.github/workflows/slack-notify.yml

```diff
@@ -0,0 +1,27 @@
+on:   
+  discussion:
+    types: [created]  
+name: Slack Notification Demo
+jobs:
+  slackNotification:
+    name: Slack Notification
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I guess this step is not directly linked to the Slack Notification? I was trying to find reference  for the 

```
PRTitle
PRUrl
PRBody
```

and I find nothing. Probably I missing something here :(

Reply:
I actually tested it again and this whole step is completely unnecessary. Let's remove it!